src/averageAges.js

```diff
@@ -54,10 +49,50 @@ function calculateWomenAverageAge(people, withChildren) {
  *
  * @return {number}
  */
+
+function calculateMenAverageAge(people, century) {
+  const men = century ? people.filter(person =>
+    Math.ceil(person.died / 100) === century).filter(findMen)
+    : people.filter(findMen);
```

Comment:
2. [CODE STYLE] - If you move your shortened return statement in arrow function to a newline, use round brackets
i'm not sure how to use it better with :
```suggestion
  const men = century ? people.filter(person => (
      Math.ceil(person.died / 100) === century).filter(findMen)
    )
    : people.filter(findMen);
```